cmd/dep/gopath_scanner.go

```diff
@@ -52,10 +52,9 @@ func (g *gopathScanner) InitializeRootManifestAndLock(rootM *dep.Manifest, rootL
 		return err
 	}
 
-	g.origM = &dep.Manifest{
-		Constraints: g.pd.constraints,
-		Ovr:         make(gps.ProjectConstraints),
-	}
+	g.origM = dep.NewManifest()
+	g.origM.Constraints = g.pd.constraints
```

Comment:
Hmmm... I thought about solutions to this but couldn't come up with anything I'm comfortable with.

Manifests are initialized once or twice per `dep` command, so it shouldn't have any impact on performance. Right?